pkg/reconciler/shared/tektonconfig/tektonconfig.go

```diff
@@ -219,6 +194,10 @@ func (r *Reconciler) ensureTargetNamespaceExists(ctx context.Context, tc *v1alph
 			}
 		}
 	} else {
+		labels := map[string]string{"operator.tekton.dev/targetNamespace": "true"}
+		if err := common.CreateTargetNamespace(labels, tc, r.kubeClientSet); err != nil {
+			return err
+		}
```

Comment:
are the lines following this logic 201-214 redundant. 201-214 is doing exactly what `common.CreateTargetNamespace()` is doing, isn't it.

Reply:
the following changes were the part of #600 hence resolved there